app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
[REQUIRED] Kindly adjust your code. Whenever you need belongs to an author instead of user do something like this:
`belongs_to :author, class_name: 'User'`